updater.go

```diff
@@ -190,7 +190,12 @@ func (u Updater) SetWebhook(path string, webhook Webhook) (bool, error) {
 
 	v := url.Values{}
 	
-	webhookURL := strings.TrimSuffix(webhook.URL, "/") + path
+	/*
+	** if URL has "/" symbol, we doesn't want to add second "/"
+	** function TrimSuffix remove "/", if it exists
+	** after this we can add "/" symbol
+	*/
+	webhookURL := strings.TrimSuffix(webhook.URL, "/") + "/" + path
```

Comment:
This does not fix anything - you're the exact character that you just trimmed. You can simply add the `/` to the path variable yourself.